File: app/src/main/java/org/jay/launchstarterexample/tasks/InitWeexTask.java
package org.jay.launchstarterexample.tasks;

import android.app.Application;
import com.taobao.weex.InitConfig;
import com.taobao.weex.WXSDKEngine;
import org.jay.appstarter.MainTask;

/**
 * 主线程执行的task
 */
public class InitWeexTask extends MainTask {

    @Override
    public boolean needWait() {
        return true;
    }

    @Override
    public void run() {
        InitConfig config = new InitConfig.Builder().build();
        WXSDKEngine.initialize((Application) mContext, config);
    }
}

File: app/src/main/java/org/jay/appstarterexample/tasks/InitWeexTask.kt
package org.jay.appstarterexample.tasks

import android.app.Application
import org.jay.appstarter.MainTask
import com.taobao.weex.InitConfig
import com.taobao.weex.WXSDKEngine

/**
 * 主线程执行的task
 */
class InitWeexTask : MainTask() {
    override fun needWait(): Boolean {
        return true
    }

    override fun run() {
        val config = InitConfig.Builder().build()
        WXSDKEngine.initialize(mContext as Application, config)
    }
}